apps/chat/src/store/overlay/overlay.reducers.ts

```diff
@@ -115,6 +118,13 @@ const selectOverlaySystemPrompt = createSelector([rootSelector], (state) => {
   return state.systemPrompt;
 });
 
+const selectOverlayNewConversationsFolder = createSelector(
+  [rootSelector],
+  (state) => {
+    return state.newConversationsFolder;
+  },
+);
+
```

Comment:
was agreed to not use createSelector for simple getters
```suggestion
const selectOverlayNewConversationsFolder = (state: RootState) =>
  rootSelector(state).newConversationsFolder;
```

Reply:
fixed